frontend/src/modules/EditZing/Components/StudentCard.tsx

```diff
@@ -115,8 +144,34 @@ const StudentCard = ({
                 position: 'absolute',
                 right: '1px',
                 top: '1px',
+                '&:hover': {
+                  transform: 'scale(1.15)',
+                },
               }}
             />
+
+            <Box
+              sx={{
+                '& :hover': {
+                  transform: 'scale(1.15)',
+                },
+              }}
+              onClick={handleOpenNotes}
+            >
+              <img
+                src={notesIcon}
+                style={{
+                  width: '24px',
+                  height: '24px',
+                  position: 'absolute',
+                  right: '-1px',
+                  top: '22px',
+                  cursor: 'pointer',
+                  display: isHovering ? '' : 'none',
+                }}
+                alt=""
```

Comment:
Why does IconButton not work here

Reply:
Yeah, I was also wondering how could I use Icon Button here? As the notes icons in the Figma file aren't actually mui icons so I just imported it as an image.

Reply:
I haven't tested it but does the Icon Button not let you use the react-component-ed version of the image as its children? In the examples it's like `<IconButton><SomeIcon /></IconButton>` and I thought it wouldn't be limited to only putting their icons as children of the component

Reply:
Yeah, it is possible. I just changed it to IconButton. Do we want the ripple effect though? I think the checkbox doesn't use it. I just used Box (with component prop set to button) as it was easier to style. Also removed transformation on hover.